config/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 	// Databricks host (either of workspace endpoint or Accounts API endpoint)
 	Host string `name:"host" env:"DATABRICKS_HOST"`
 
+	ClusterId string `name:"cluster_id" env:"DATABRICKS_CLUSTER_ID"`
```

Comment:
Please use `ClusterID` to match the style of the other fields.

Also see: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

We can't enforce this everywhere, e.g. for generated code, but should try to stick to it where it makes sense.